tests/integration/docusaurus/setup/configuring_data_docs/how_to_host_and_share_data_docs_on_gcs.py

```diff
@@ -7,9 +7,14 @@
 
 context = ge.get_context()
 
+# NOTE: The following code is only for testing and depends on an environment
+# variable to set the gcp_project. You can replace the value with your own
+# GCP project information
+gcp_project = os.environ.get("GE_TEST_GCP_PROJECT")
```

Comment:
Should this have error checking as well? What if I didn't set my env var?

Reply:
This applies to any other usages of env var retrieval herein.

Reply:
Good point